frontend/lib/src/components/shared/Radio/Radio.tsx

```diff
@@ -46,6 +47,30 @@ export interface Props {
   help?: string
 }
 
+function getRadioInnerSizes(theme: Theme): [string, string] {
+  // If checked, the radio inner circle should fill 37.5% of the total radio size.
+  // If not checked, it should show a border of spacing.threeXS.
+
+  // However, fractional pixels could cause the radio border to look uneven. This happens
+  // when (checkbox - threeXS) in rem is not an integer number of pixels. To avoid this,
+  // we round the number converted from rem to pixels then add back the unit.
+  const checkboxSize = parseFloat(theme.sizes.checkbox.replace("rem", ""))
+  const threeXSSpacing = parseFloat(theme.spacing.threeXS.replace("rem", ""))
```

Comment:
nit: I think we don't even need the `.replace("rem", "")` here. `parseFloat` can also handle something with a prefix (e.g. "1.2rem").